### PR TITLE
Removed '>>>' and '...' from examples and docs and improved some indentation.

### LINKED ISSUE #758: Remove `>>>` and `...` from docs and examples, and make code snippets easy to use
### Bug

TODO

### Environment

TODO

### Minimal Reproducible Example

TODO

### Additional

TODO

### PR DESCRIPTION
**Pull request by RaghavvGupta**
## Description

Removed '>>>' and '...' from examples and docs and improved some indentation. Please guide whether any changes are required in the documentation or in the PR. (Fix #758 ) 

## Docs

-   [x] Docs updated? What were the changes:

### CODE REVIEW

File: docs/annotators.md [hunk 1]
```diff
@@ -194,16 +194,16 @@ comments: true
 === "Mask"
 
     ```python
-    >>> import supervision as sv
+    import supervision as sv
 
-    >>> image = ...
-    >>> detections = sv.Detections(...)
+    image =
+    detections = sv.Detectio
```
Comment (onuralpszr) [inline docs/annotators.md:200]: Fix typo

File: docs/annotators.md [hunk 2]
```diff
@@ -194,16 +194,16 @@ comments: true
 === "Mask"
 
     ```python
-    >>> import supervision as sv
+    import supervision as sv
 
-    >>> image = ...
-    >>> detections = sv.Detections(...)
+    image =
```
Comment (onuralpszr) [inline docs/annotators.md:199]: Type like 'image = ...'   or place image file name or load via image cv2 for making more readable. Same for others

**Comment by onuralpszr** [comment]
@RaghavvGupta  remove .DS_Store file. It should never come to PR please rebase/amend your commit and force push it so it should be gone and It should not create extra commit message for that.

**Comment by RaghavvGupta** [comment]
Hey @onuralpszr pls tell me if there are any other changes required

**Comment by SkalskiP** [comment]
Hi @RaghavvGupta 👋🏻!  We need to apply the same change to all our docs.

**Comment by RaghavvGupta** [comment]
Hey @SkalskiP , sure, I will make the changes! 😀

**Comment by RaghavvGupta** [comment]
Sure, I understand @onuralpszr! I apologize for the delay. Loved working on the issue and collaborating with you guys, you love to contribute more!

**Comment by SkalskiP** [comment]
@RaghavvGupta I'm working on our roadmap for the next 3 months. You should be able to pick up something interesting soon ;)

**Pull request merged**